### PR TITLE
Fix example issues

### PR DESCRIPTION
**Pull request by anupama-pathirage**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/functions-with-rest-parameter/functions_with_rest_parameter.bal [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import ballerina/io;
 
-// This function takes one required parameter and one rest parameter of type 
-// string. A function can have only one rest parameter.
+// This function takes one required parameter, one defaultable parameter and
+// one rest parameter of type string. A function can have only one rest parameter.
```
Comment (samgnaniah) [inline examples/functions-with-rest-parameter/functions_with_rest_parameter.bal]: Shall we change 'rest' to 'REST'. Apply elsewhere if applicable.

File: examples/foreach/foreach.description [hunk 2]
```diff
@@ -1,2 +1,2 @@
 // The 'Foreach' statement is a looping construct that traverses through the items of a collection of data such as
-// arrays, maps, json, and xml.
+// arrays, maps, json, xml and table.
```
Comment (samgnaniah) [inline examples/foreach/foreach.description]: Shall we change 'json' to 'JSON'. Same for XML?

File: examples/table-queries/table_queries.bal [hunk 3]
```diff
@@ -40,114 +40,77 @@ type PersonPublicProfile record {
...
+    };
 
     // This prints the `Order` table content.
-    io:print("\nThe orderTable: ");
-    io:println(orderTable);
+    printTable(queryStmt, "The orderTable: ", orderTable);
 
     // Querying for a table always returns a new in-memory table.
 
     // 1. This queries for all the records in a table and returns them as another in-memory table.
```
Comment (samgnaniah) [inline examples/table-queries/table_queries.bal]: Can we remove the numbers? We do not seem to follow this format for other examples. WDYT?

File: examples/functions-with-rest-parameter/functions_with_rest_parameter.bal (hunk 1 above)
Comment (anupama-pathirage) [reply examples/functions-with-rest-parameter/functions_with_rest_parameter.bal]: This is rest parameter nothing related to REST(REpresentational State Transfer). So kept as it is.

**Review by samgnaniah — APPROVED** [review]

**Review by hasithaa — COMMENTED** [review]
Table literal has to be updated with new syntax.

**Review by hasithaa — APPROVED** [review]

**Pull request merged**